.vscode/scripts/vss.json

```diff
@@ -0,0 +1,6177 @@
+{
```

Comment:
If vss is part of the development environment, it should be part of .devcontainer. It's not specific to vs code?!

Reply:
This part of the repo is synced with Python, I will pick this up as finding and address it in all repos simultaneously.